src/openrct2-android/app/src/main/CMakeLists.txt

```diff
@@ -131,7 +131,7 @@ include_directories(SYSTEM "${CMAKE_BINARY_DIR}/libs/include/freetype2")
 include_directories(SYSTEM "${CMAKE_BINARY_DIR}/libs/include/SDL2")
 
 # now build app's shared lib
-add_definitions(-DDISABLE_HTTP -DDISABLE_DISCORD_RPC -DDISABLE_HTTP -DDISABLE_NETWORK -DDISABLE_OPENGL -DGL_GLEXT_PROTOTYPES -D__STDC_LIMIT_MACROS -DNO_TTF -DSDL_MAIN_HANDLED)
+add_definitions(-DDISABLE_HTTP -DDISABLE_DISCORD_RPC -DDISABLE_HTTP -DDISABLE_NETWORK -DDISABLE_FLAC -DDISABLE_VORBIS -DDISABLE_OPENGL -DGL_GLEXT_PROTOTYPES -D__STDC_LIMIT_MACROS -DNO_TTF -DSDL_MAIN_HANDLED)
```

Comment:
Do we want to disable this on Android? (I don’t know how difficult it is to add this to Android, perhaps @janisozaur can tell?)

Reply:
Trying to add dependencies is very convoluted given all the different platforms you need to account for. I don't know how the dependencies on Android work, and it looks like it disables many of the other dependencies we also use. I didn't want Android to hold back this PR.

Reply:
#17349